packages/scheduled-payments-client/package.json

```diff
@@ -70,11 +72,13 @@
     "ember-cli-typescript": "^5.1.1",
     "ember-cli-typescript-blueprints": "^3.0.0",
     "ember-fetch": "^8.1.1",
+    "ember-focus-trap": "^1.0.1",
```

Comment:
I don't know if there's some ember limitation, but shouldn't boxel, handled all required deps in it's package.json, so we don't need to individually add the package on each client ?

Reply:
You are generally right. There is a funky embroider/addon bug that necessitates this.

Reply:
https://github.com/embroider-build/embroider/issues/1175